nequip/model/_scaling.py

```diff
@@ -11,25 +11,44 @@
 
 
 def RescaleEnergyEtc(
+    model: GraphModuleMixin, config, dataset: AtomicDataset, initialize: bool
+):
+
+    return GeneralRescale(
+        model=model,
+        config=config,
+        dataset=dataset,
+        initialize=initialize,
+        module_prefix="global_rescale",
+        default_scale=f"dataset_{AtomicDataDict.FORCE_KEY}_rms"
+        if AtomicDataDict.FORCE_KEY in model.irreps_out
+        else f"dataset_{AtomicDataDict.TOTAL_ENERGY_KEY}_std",
+        default_shift=None,
+        default_scale_keys=AtomicDataDict.ALL_ENERGY_KEYS,
+        default_shift_keys=AtomicDataDict.TOTAL_ENERGY_KEY,
+        default_related_keys=[AtomicDataDict.PER_ATOM_ENERGY_KEY],
+    )
+
+
+def GeneralRescale(
```

Comment:
Maybe we call this `GlobalRescale` to hammer home the difference vs PerSpecies etc.?

Reply:
good idea. pushed.